core/src/main/resources/jenkins/monitor/JavaVersionRecommendationAdminMonitor/message.jelly

```diff
@@ -26,25 +26,13 @@ THE SOFTWARE.
 <j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout" xmlns:f="/lib/form">
     <j:invokeStatic var="currentRuntimeJavaVersion" className="jenkins.util.java.JavaUtils" method="getCurrentRuntimeJavaVersion" />
     <div class="alert alert-danger">
-        <h2>${%Recommended_Java_Version_Heading(currentRuntimeJavaVersion)}</h2>
-        ${%Recommended_Java_Version(currentRuntimeJavaVersion)}
-      <style>
-        .manage-messages .alert #JavaVersionRecommendationAdminMonitor span,
-        .manage-messages .alert #JavaVersionRecommendationAdminMonitor,
-        .am-container .am-message .alert #JavaVersionRecommendationAdminMonitor span {
-          margin-left: 0 !important;
-        }
-      </style>
-        <form id="JavaVersionRecommendationAdminMonitor" method="post" action="${rootURL}/${it.url}/act" name="${it.id}" style="float: initial">
-          <div>
+        <form id="JavaVersionRecommendationAdminMonitor" method="post" action="${rootURL}/${it.url}/act" name="${it.id}">
             <f:submit name="yes" value="${%More Info}"/>
             <l:isAdmin>
-              <input type="submit"
-                     name="no"
-                     value="${%Dismiss}"
-                     class="submit-button" />
+                <f:submit value="${%Ignore}" name="no" clazz="submit-button"/>
```

Comment:
Does it need the submit-button class? Nothing stood out for a quick CMD+SHIFT+F that I did

Reply:
Good catch! Works well without too 👍